pkg/utils/dangerous_secret_names.go

```diff
@@ -0,0 +1,40 @@
+package utils
```

Comment:
Commit message: "Add functionality to check for dangerous secret names" vs "Check for dangerous secret names in `run` command"